packages/php/email-editor/src/Container.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\EmailEditor;
+
+class Container {
+  protected array $services = [];
+  protected array $instances = [];
+
+  public function set(string $name, callable $callable): void {
+    $this->services[$name] = $callable;
+  }
+
+  /**
+   * @template T
+   * @param class-string<T> $id
```

Comment:
Oh! I noticed it should be `$name` in this case. Sorry! I copied the annotation from the current free container where we use $id as parameter name.

```suggestion
   * @param class-string<T> $name
```